tripmining/model/trip.py

```diff
@@ -327,6 +363,24 @@ def to_json_with_checkins(self):
     }},
 """
 
+    def __get_transitions(self) -> list:
+        transitions = list()
+        previous_block = self.blocks[0]
+        # add transition for every block change
+        for block in self.blocks[1:]:
+            current_transition = Transition(previous_block.checkins[0].location, block.checkins[0].location,
+                                            block.checkins[0].date - previous_block.checkins[0].date)
+            previous_block = block
+            transitions.append(current_transition)
+        return transitions
+
+    def __get_max_transition_time(self) -> float:
```

Comment:
I'd prefer to use the built-in `max()` function: https://docs.python.org/3/library/functions.html#max

Reply:
Accepted and fixed in the latest PR